live-examples/css-examples/border-color.html

```diff
@@ -0,0 +1,37 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="border-color">
+
+    <div class="example-choice" initial-choice="true">
+        <pre><code id="example_one" class="language-css">border-color: red;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_one">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_two" class="language-css">border-color: red #4281de;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true" data-clipboard-target="#example_two">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">border-color: red rgb(240,30,50,.7) green;</code></pre>
```

Comment:
`red` and `rgb(240,30,50,.7)` are too similar for the difference to be obvious, especially with this quite narrow border. I do like the fact that this demonstrates the use of `rgb()`, but it would be better to choose a more contrasting value for it.